<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_principal</class>
 <widget class="QMainWindow" name="ventana_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Sitka&quot;;  background-color: rgb(249, 247, 245)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_cerrar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>750</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(159, 183, 190)</string>
    </property>
    <property name="text">
     <string>Cerrar Session</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;


background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>üì∑ Cargar Imagenes</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="saludo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Hola...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_procesar_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>‚öôÔ∏è Procesar Imagenes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_ver_daton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>üìä Ver datos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_pricipal.qrc"/>
 </resources>
 <connections/>
</ui>
